N_bonacci.dart

```diff
@@ -0,0 +1,21 @@
+void N_bonacci(int n, int m) {
+  var v = new List(m);
```

Comment:
You could return `v` and test it.
```suggestion
import 'package:test/test.dart';

List N_bonacci(int n, int m) {
  List v = new List(m);
```